Functions/xASL_Iteration.m

```diff
@@ -328,11 +328,18 @@
         end
         
         % Some feedback about this iteration (after opening diary log)
+                
         if ~AlreadyProcessed
             fprintf('\n%s\n',repmat('+',1,72)); % just draw a separator line
-            fprintf('%s\n',['Starting ' x.MUTEXID ' at ' datestr(now)]);
+            [StartI, EndI] = regexp(diaryFileEx, 'ASL_\d','ONCE'); %to find the name of the session inside diaryFileEx: ASL_with any digit after
```

Comment:
Nice! Does this reading of the diary file always work? Also in compiled/deployed mode etc? Shouldn't the current session be easier to find in the current table, e.g. `x.SESSION` or `x.SESSIONDIR` or something?